packages/api/src/schema/schema.yaml

```diff
@@ -425,6 +507,36 @@ components:
           type: string
           readOnly: true
           example: abc123
+        firstName:
+          type: string
+          example: Joe
+        lastName:
+          type: string
+          example: Smith
+        organization:
+          type: string
+          example: Livepeer
+        phone:
+          type: string
+          example: 2034212122
+        stripeProductId:
+          type: string
+          example: prod_0
+        stripeCustomerId:
+          type: string
+          example: cus_Jv6KvgT0DCH8HU
```

Comment:
This should presumably have `unique: true`, if I understand the flow correctly things would break pretty hard if two users have the same stripe customer id

Reply:
Not that it should ever happen... but it doesn't hurt to have the DB to enforce that too. Bonus: that unique index will make lookups by customer ID in faster than `O(every single user)` time

Reply:
Fixed in [04ff25](https://github.com/livepeer/livepeer-com/pull/25/commits/04ff2537a397629316e3ccef0ffba6a97665ad0a)